client/js/backstage.js

```diff
@@ -157,29 +157,29 @@ var BackstageView = Backbone.View.extend({
         this.$openButton    = this.$el.find('.silky-bs-open-button')
         this.$saveButton    = this.$el.find('.silky-bs-save-button')
         this.$saveAsButton  = this.$el.find('.silky-bs-save-as-button')
-		
+
 		this.main = new BackstagePlaces({ el: ".silky-bs-main", model: this.model })
 	},
 	activate : function() {
-	
+
 		this.$el.fadeIn(100)
-		
```

Comment:
ah! ok. so you used a "fix indentation" function or the like? it fixed all the tabs, but it *also* removed all the white space from empty lines (and you assumed it was removing tabs). so it sounds like your text editor is fine.